packages/common/src/utils/regex/RegexUtils.tsx

```diff
@@ -23,4 +23,9 @@ export const RegexUtils = {
       />
     );
   },
+  // Case-insensitive match of partial string -- same as SQL "LIKE"
+  matchSubstring: (substring: string, testString: string) => {
+    const matcher = new RegExp(substring, 'i');
+    return matcher.test(testString);
+  },
```

Comment:
I couldn't find a simple partial string matcher anywhere already -- let me know if I've missed an existing one.

Reply:
no, there isn't one! because the filtering is all server-side we haven't had much need for client side utils.
You could add a test - but I think not worth it yet.